key/src/platform_address.rs

```diff
@@ -141,6 +141,12 @@ impl From<&'static str> for PlatformAddress {
     }
 }
 
+impl Into<Address> for PlatformAddress {
```

Comment:
FYI, it will conflict with https://github.com/CodeChain-io/codechain/pull/578/commits/86328a85584a0b2af399b6818d7b49241e75f069